composer.json

```diff
@@ -29,15 +29,17 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.3 || ^8.0",
```

Comment:
Please recheck to list in [related issue report](https://github.com/mezzio/mezzio-authentication/issues/5):

> Modify `composer.json` to provide support for PHP 8.0 by adding the constraint `~8.0.0`

Thanks in advance!

Reply:
@froschdesign, fixed.